Gopkg.lock

```diff
@@ -0,0 +1,159 @@
+  revision = "ddbde93f65ab0692e54ed8a5ad325fa2e8af4daa"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/decred/dcrwallet"
+  packages = ["apperrors","chain","internal/zero","snacl","wallet/txrules","wallet/udb","walletdb"]
+  revision = "e3aa7eafac265c77c59cf7874307a1ea28b80ee0"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/dgrijalva/jwt-go"
+  packages = ["."]
+  revision = "a539ee1a749a2b895533f979515ac7e6e0f5b650"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/go-gorp/gorp"
+  packages = ["."]
+  revision = "659d466222f6a201e44cfeff05813fa6be8409cd"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/go-sql-driver/mysql"
+  packages = ["."]
+  revision = "be22b3051b3467094e9908389d900e74f72cf08f"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/golang/protobuf"
+  packages = ["proto","ptypes","ptypes/any","ptypes/duration","ptypes/timestamp"]
+  revision = "ae59567b9aab61b50b2590679a62c3c044030b61"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/gorilla/context"
+  packages = ["."]
+  revision = "08b5f424b9271eedf6f9f0ce86cb9396ed337a42"
+
+[[projects]]
+  name = "github.com/gorilla/securecookie"
+  packages = ["."]
+  revision = "667fe4e3466a040b780561fe9b51a83a3753eefc"
+  version = "v1.1"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/gorilla/sessions"
+  packages = ["."]
+  revision = "b61c93cb7f67533c8bfbb5ea450efc07e5833c5e"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/haisum/recaptcha"
+  packages = ["."]
+  revision = "7d3b8053900e7a0f549a11e800649a5fa950700f"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/jrick/logrotate"
+  packages = ["rotator"]
+  revision = "a93b200c26cbae3bb09dd0dc2c7c7fe1468a034a"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/zenazn/goji"
+  packages = ["graceful","graceful/listener","web","web/middleware","web/mutil"]
+  revision = "c05078ca81941f8e801bba3ddc0e6b86b7fdc893"
+
+[[projects]]
+  branch = "master"
+  name = "golang.org/x/crypto"
+  packages = ["bcrypt","blowfish","nacl/secretbox","pbkdf2","poly1305","ripemd160","salsa20/salsa","scrypt"]
+  revision = "7d9177d70076375b9a59c8fde23d52d9c4a7ecd5"
+
+[[projects]]
+  branch = "master"
+  name = "golang.org/x/net"
+  packages = ["context","http2","http2/hpack","idna","internal/timeseries","lex/httplex","trace"]
+  revision = "8351a756f30f1297fe94bbf4b767ec589c6ea6d0"
+
+[[projects]]
+  branch = "master"
+  name = "golang.org/x/text"
+  packages = ["collate","collate/build","internal/colltab","internal/gen","internal/tag","internal/triegen","internal/ucd","language","secure/bidirule","transform","unicode/bidi","unicode/cldr","unicode/norm","unicode/rangetable"]
+  revision = "1cbadb444a806fd9430d14ad08967ed91da4fa0a"
+
+[[projects]]
+  branch = "master"
+  name = "google.golang.org/genproto"
+  packages = ["googleapis/rpc/status"]
+  revision = "1e559d0a00eef8a9a43151db4665280bd8dd5886"
+
+[[projects]]
+  branch = "master"
```

Comment:
Please use a stable branch of gRPC.  i.e.:

https://github.com/grpc/grpc-go/tree/v1.6.x